tests/app/main/helpers/test_frameworks.py

```diff
@@ -638,3 +641,131 @@ def test_calls_logger_if_provided(self, abort):
                 extra={'error': 'An error from the API (status: 404)', 'framework_slug': 'g-cloud-7'},
             )
         ]
+
+
+class TestEnsureApplicationCompanyDetailsHaveBeenConfirmed(BaseApplicationTest):
+    def setup_method(self, method):
+        super().setup_method(method)
+
+        from app import data_api_client
+        self.data_api_client_mock = mock.Mock(spec_set=data_api_client)
+        self.decorator = EnsureApplicationCompanyDetailsHaveBeenConfirmed(self.data_api_client_mock)
+        # self.login()
```

Comment:
Can this go?

Reply:
Also, I don't see `self.decorator` being used anywhere else. Does it need to be part of the setup like this?

Reply:
yeah these are probably both hangovers from intermediate work, will remove